### DISCUSSION TITLE
Breadcrumbs links are not working

### DISCUSSION THREAD

**Discussion by PatriziaGm** [udayvunnam/xng-breadcrumb#164] [Q&A]
Hello, 
I installed the library yesterday and I found amazing that with a few lines the BreadCrumb showed correctly on the page. 
I noticed then that the links are not working: it is very strange, if I subscribe the Observable this.breadcrumbService.breadcrumbs$ I see the data in the console and  I get also the correct routeLink.
But in the template the route is not showing: 

a role="button" rel="noopener noreferrer" class="xng-breadcrumb-link" ng-reflect-ng-class="[object Object]" target="_self" tabindex="0"

I' using version ^9.0.0.
What am I doing wrong? I tried already a lot of possibilities, with personal label, with alias... but nothing works.
Looking forward for your answer
Pat

**Comment by udayvunnam** [comment]
Hi @PatriziaGm can you check the version compatiability guide and see you're using correct version of angular+breadcrumb https://github.com/udayvunnam/xng-breadcrumb/blob/main/docs/angular-version-compatibility.md

Also, https://github.com/udayvunnam/xng-breadcrumb/tree/main/apps has number of examples
